piwik-sdk/src/main/java/org/piwik/sdk/Piwik.java

```diff
@@ -68,6 +68,16 @@ public synchronized Tracker newTracker(@NonNull String trackerUrl, int siteId) t
     }
 
     /**
+     * Configure Piwik to receive more than 5 custom variable
+     * http://piwik.org/faq/how-to/faq_17931/
+     * Changing this limit will affect all Tracker instances
+     * @param maxVariables
+     */
+    public void setMaxCustomVariables(int maxVariables){
+        CustomVariables.MAX_VARIABLES = maxVariables;
```

Comment:
What about removing the limit from the custom variables object and when submitting the trackme, the trackers trims the custom variables to the allowed amount before handing it to the dispatcher?